src/components/spinner/Spinner.tsx

```diff
@@ -0,0 +1,17 @@
+import type { DataSpinner } from './Spinner.types';
+
+const list = Array.from(new Array(12).fill(null), (_, idx) => idx + 1);
+const divList = list.map((value) => <div key={value}/>);
+
+export default function Spinner(props: DataSpinner) {
```

Comment:
This spinner is screaming "What year it is?!" xD Here you go - https://github.com/n3r4zzurr0/svg-spinners - you are welcome xD